nixpkgs_review/review.py

```diff
@@ -228,6 +228,7 @@ def start_review(
         pr: Optional[int] = None,
         post_result: Optional[bool] = False,
     ) -> None:
+        del os.environ["NIXPKGS_CONFIG"]
```

Comment:
For multiple reviews at the same time this might crash nixpkgs-review if tries to remove a non-existing key:

```suggestion
        os.environ.pop("NIXPKGS_CONFIG", None)
```